.github/workflows/lint.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
           python-version: '3.10'
 
       - name: Install packages
-        run: pip install -r requirements/dev.txt
+        run: make install-dev
```

Comment:
Do we have a rule for this? I couldn't seem to run `install-dev`, maybe you want `make venv`?

Reply:
We can also refactor to mirror the `uv` based approach from here:

https://github.com/populationgenomics/cpg-flow/pull/1/files#diff-4b122024a3a28ded65da76a2f1bface1f3a27328374438d1298d25585fe0603b

Reply:
Yup just finished